src/modules/common/components/bars/sideBar/index.js

```diff
@@ -47,7 +53,14 @@ const MenuLink = ({ data, pathname, sideBarExpanded, disabled }) => {
       activeClassName={styles.selected}
       exact={routes[data.id].exact}
     >
-      <Inner data={data} pathname={pathname} sideBarExpanded={sideBarExpanded} />
+      <Inner
+        data={data}
+        pathname={pathname}
+        sideBarExpanded={sideBarExpanded}
+        notification={
```

Comment:
since its a boolean type, I think the naming here should be 

```suggestion
        hasNotification={
```

Reply:
Fixed